app/views/badges/index.html.erb

```diff
@@ -14,15 +14,3 @@
 <div class="badge_list">
   <%= render 'badges/badge_list' %>
```

Comment:
I somehow made this partial break. Not sure if it's your fault but here's a comment anyways :^)